docs/changes/CHANGES

```diff
@@ -1,12 +1,47 @@
-MoinMoin Version History
-========================
+MoinMoin 2 Version History
+==========================
 
 Please note: It is recommended that existing wiki's be upgraded to the
 latest moin 1.9.x release before converting to Moin 2. However, this may
 not be a necessary step as the 1.9.x file structure has not changed recently.
 
-Version 2.0.0alpha
-------------------
+Version 2.0.0beta 2024-??-??
+----------------------------
+This is an unstable beta release not suitable for a production wiki,
+test carefully and report new issues and feature requests on the issue tracker:
+https://github.com/moinwiki/moin/issues.
+
+Fixes
+~~~~~
+* added config option to enable or disable contenttypes
+* mass source code changes using pyupgrade to upgrade syntax to 3.9
+* removed unused imports
+* create SECURITY.md
+* generate readthedocs content using Python 3.10
```

Comment:
For my projects I now tend to only include stuff that is somehow "interesting" for users of the software or package maintainers and try to leave away minor details (one person is writing this, but potentially many people are reading this - saves them time).